README.md

```diff
@@ -130,17 +130,34 @@ ng build
 
 The build artifacts will be stored in the `dist/` directory.
 
-### Environments
+### Build targets and environment files
 
-At build time, the `src/app/environment.ts` will be replaced by either
-`config/environment.dev.ts` or `config/environment.prod.ts`, depending on the
-current cli environment. The resulting file will be `dist/app/environment.ts`.
+A build can specify both a build target (`development` or `production`) and a 
+environment file to be used with that build.
 
-Environment defaults to `dev`, but you can generate a production build via
-the `-prod` flag in either `ng build -prod` or `ng serve -prod`.
+At build time, `src/app/environments/environment.ts` will be replaced by either
+`src/app/environments/environment.{NAME}.ts` where `NAME` is the argument 
+provided to the `--environment` flag.
+
+These options also apply to the serve command and default to `development` and 
+`dev` respectively.
+If you only specify the build target, the environment file will default to `dev` 
+for `development` and `prod` for `production`. 
```

Comment:
Remove the "and default to `development` and `dev` respectively" from the first text:

```markdown
These options also apply to the serve command. If you do not pass a value for `environment`,
it will default to `dev` for `development` and `prod` for `production`.
```